apps/daily-scrum/src/lib/cache.ts

```diff
@@ -0,0 +1,69 @@
+import { unstable_cache } from "next/cache";
+import { cache } from "react";
+
+function persist<T extends any[], R>(
+  callback: (...args: T) => Promise<R>,
+  functionName: string
+) {
+  return (...args: T) => {
+    const key = `${functionName}(${args.join(",")})`;
+    return unstable_cache(
+      async (...args: any[]) => {
+        return callback(...(args as T));
+      },
+      [key],
+      {
+        tags: [key],
+      }
+    )(...args);
+  };
+}
```

Comment:
A function that calls unstable_cache for persistence and assigns a tag as a revalidation key.